src/extension/background-script/bitcoin/index.ts

```diff
@@ -0,0 +1,84 @@
+import * as btc from "@scure/btc-signer";
+import Mnemonic from "~/extension/background-script/mnemonic";
+import { BitcoinAddress, BitcoinNetworkType } from "~/types";
+
+const BTC_TAPROOT_DERIVATION_PATH = "m/86'/0'/0'/0";
+const BTC_TAPROOT_DERIVATION_PATH_REGTEST = "m/86'/1'/0'/0";
+
+class Bitcoin {
+  readonly networkType: BitcoinNetworkType;
```

Comment:
Clear memoized objects when the network setting in the account setting is changed.